common/src/main/resources/messages/messages.properties

```diff
@@ -685,6 +685,13 @@ datalake.verticalscale.datalake.in.progress=Vertical scale Data Lake is in progr
 datalake.verticalscale.datalake.failed=Vertical scale Data Lake failed {0}
 datalake.verticalscale.failed=Vertical scale failed on Data Lake {0} please use a different instance type or retry the operation
 
+datalake.diskupdate.validation.in.progress=Validation for disk update Data Lake is in progress
+datalake.diskupdate.validation.failed=Validation for disk update Data Lake failed {0}
+datalake.diskupdate.datalake.in.progress=Disk update Data Lake is in progress
+datalake.diskupdate.datalake.failed=Disk update Data Lake failed
```

Comment:
`Disk update Data Lake` please fix it, I guess you mean `Disk update on Data Lake`

Reply:
done